sdk/storage/azure-storage-queue/azure/storage/queue/_shared/base_client_async.py

```diff
@@ -81,5 +82,6 @@ def _create_pipeline(self, credential, **kwargs):
             config.retry_policy,
             config.logging_policy,
             AsyncStorageResponseHook(**kwargs),
+            DistributedTracingPolicy(),
```

Comment:
This doesn't need to be an async version of the policy?

Reply:
Additionally - I see you've added the policy to the async pipeline, but haven't decorated the async clients?

Reply:
just decorated them